test/organization.js

```diff
@@ -52,43 +52,43 @@ describe("Organization", () => {
     const contracts = await helpers.contractsForTest();
     // a new organization comes with three known schemes
     assert.equal((await organization.schemes()).length, 3);
-    let scheme = await organization.scheme('GlobalConstraintRegistrar');
+    let scheme = await organization.scheme("GlobalConstraintRegistrar");
     assert.equal(
       scheme.address,
       contracts.allContracts.GlobalConstraintRegistrar.address
     );
     assert.isTrue(!!scheme.contract, "contract must be set");
-    scheme = await organization.scheme('SchemeRegistrar');
+    scheme = await organization.scheme("SchemeRegistrar");
     assert.equal(
       scheme.address,
       contracts.allContracts.SchemeRegistrar.address
     );
     assert.isTrue(!!scheme.contract, "contract must be set");
-    scheme = await organization.scheme('UpgradeScheme');
+    scheme = await organization.scheme("UpgradeScheme");
     assert.equal(
       scheme.address,
       contracts.allContracts.UpgradeScheme.address
     );
     assert.isTrue(!!scheme.contract, "contract must be set");
 
     // now we add another known scheme
-    await proposeContributionReward(organization, accounts);
+    await proposeContributionReward(organization);
 
     assert.equal((await organization.schemes()).length, 4);
     // TODO: the organizaiton must be registered with the scheme before the next works
-    // assert.equal((await organization.scheme('ContributionReward')).address, contracts.allContracts.ContributionScheme.address);
+    // assert.equal((await organization.scheme("ContributionReward")).address, contracts.allContracts.ContributionScheme.address);
```

Comment:
please remove commented code

Reply:
if this is a partly done test then seems ok to leave to me for now